Results 3/N (Total 4/N) (#163)
AlexsLemonade/OpenPBTA-manuscript: content/04.results.md

Stub for the process for contributing analytical code (*Figure S1* and *STAR Methods*) and to the manuscript

### Molecular Subtyping of OpenPBTA CNS Tumors

> Over the past two decades, together with the World Health Organization (WHO), experts in neurooncology have iteratively redefined the classifications of central nervous system (CNS) tumors (@doi: 10.1093/jnen/61.3.215; @doi:10.1007/s00401-007-0243-4). 

Reviewer: ```suggestion
Over the past two decades, together with the World Health Organization (WHO), experts in neurooncology have iteratively redefined the classifications of central nervous system (CNS) tumors [@doi: 10.1093/jnen/61.3.215; @doi:10.1007/s00401-007-0243-4]. 
```

> More recently, in 2016 and 2021 (@doi:10.1007/s00401-016-1545-1; @doi:10.1093/neuonc/noab106), molecular subtypes have been integrated into these entities. 

Reviewer: ```suggestion
More recently, in 2016 and 2021 [@doi:10.1007/s00401-016-1545-1; @doi:10.1093/neuonc/noab106], molecular subtypes have been integrated into these entities. 
```

The Children’s Brain Tumor Tissue Consortium (CBTTC), currently the Children’s Brain Tumor Network (CBTN), opened its protocol for brain tumor and matched normal sample collection in 2011 and as such, the majority of the samples within the OpenPBTA dataset lack molecular subtype annotation. 

> In the absence of methylation arrays for tumor classification, we utilized key genomic features of entities described by the WHO in 2016, as well as Ryall and colleagues (@doi:10.1016/j.ccell.2020.03.011), coupled with clinician and pathologist review to subtype 64% (1,281/2,007) of tumor biospecimens with high confidence (*Supplemental Table S1*).

Reviewer: ```suggestion
In the absence of methylation arrays for tumor classification, we utilized key genomic features of entities described by the WHO in 2016, as well as Ryall and colleagues [@doi:10.1016/j.ccell.2020.03.011], coupled with clinician and pathologist review to subtype 64% (1,281/2,007) of tumor biospecimens with high confidence (*Supplemental Table S1*).
```

> Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (eg: an ETMR entered as a medulloblastoma) and mis-identified specimens (eg: ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using modern terms (eg: primitive neuro-ectodermal tumor (PNET) diagnoses), and discover rarer tumor entities within the OpenPBTA (eg: H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion).

Reviewer: ```suggestion
Importantly, this collaborative molecular subtyping process allowed us to identify data entry errors (e.g., an ETMR entered as a medulloblastoma) and mis-identified specimens (e.g., Ewing sarcoma sample labeled as a craniopharyngioma), update diagnoses using modern terms (e.g., primitive neuro-ectodermal tumor [PNET] diagnoses), and discover rarer tumor entities within the OpenPBTA (e.g., H3-mutant ependymoma, meningioma with _YAP1::FAM118B_ fusion).
```

> *Table 1* {#tbl:subtypes} lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 476), high-grade gliomas (N = 314), glialneuronal tumors (N = 14), medulloblastomas (N = 190), chordomas (N = 12), ependymomas (N = 65), other embryonal (N = 39), craniopharyngiomas (N = 51), neurocytoma (N = 6), gangliogliomas (N = 93), and ewing sarcoma (N = 21) (for methods, see *STAR Methods* and *Supplemental Figure S1*). 

Reviewer: ```suggestion
*Table 1* {#tbl:subtypes} lists the subtypes we defined within the OpenPBTA, comprising of LGATs (N = 476), high-grade gliomas (N = 314), glialneuronal tumors (N = 14), medulloblastomas (N = 190), chordomas (N = 12), ependymomas (N = 65), other embryonal (N = 39), craniopharyngiomas (N = 51), neurocytoma (N = 6), gangliogliomas (N = 93), and ewing sarcoma (N = 21). For methods, see *STAR Methods* and *Supplemental Figure S1*. 
```
